src/printer/index.js

```diff
@@ -189,10 +150,29 @@ function printIf(path, print) {
     indent(concat([hardline, printBody(path, print)]))
   ];
 
-  const orElse = printOrElse(path);
+  while (n.orelse && n.orelse.length === 1 && n.orelse[0].ast_type === "If") {
```

Comment:
nit: wouldn't hurt to have a comment or two in here :) Maybe something along the lines of this?
```python
# "Handle the nested `elif`, which are represented as `else: if...` in the AST."
```
Up to you though :)